lib/rules/max-nesting-depth/index.js

```diff
@@ -120,10 +132,24 @@ const rule = (primary, secondaryOptions) => {
 		 * @param {string[]} selectors
 		 * @returns {boolean}
 		 */
-		function containsIgnoredPseudoClassesOnly(selectors) {
-			if (!(secondaryOptions && secondaryOptions.ignorePseudoClasses)) return false;
+		function containsIgnoredPseudoClassesOrRulesOnly(selectors) {
```

Comment:
nit

What about `hasIgnoredSelector`?

Reply:
Hm, in this case it would have to be something like `hasOnlyIgnoredSelectors`, and there are some other functions in this rule that are also `contains*`. 

I do feel like these could be named better - how about I follow up with another PR to quickly make the names consistent? The nice thing about that is that it's not a user-facing change (so there's no urgency, etc.).

Reply:
> how about I follow up with another PR

No need it was really minor.